src/modules/commentsCounter.js

```diff
@@ -0,0 +1,16 @@
+function plusPlus(counter) {
```

Comment:
- [ ] I can see you have used the `function` keyword everywhere to create the Javascript functions. However, according to the requirements, the project should use ES6 arrow functions. Let's follow the requirements for your all Javascript functions in the project.

Reply:
[OPTIONAL] I'm sorry this has been marked as mandatory. Let's leave it optional, however, I do recommend making use of the ES6 syntax as much as possible, first, because it's more concise, two, because it will help you get used to it if you have been working with JS for a while and nowadays it's the trend.

![image](https://user-images.githubusercontent.com/79114233/147358996-f1c8a939-9096-43e7-a3ef-eee13526ffed.png)

Reply:
Thanks @D4nielJ for your review, i can see what you are pointing to :+1:,

Its actually a personal preference where do you wanna use arrow functions,
for my partner and I, we agreed to use explicit `function` keyword whenever we defining a function, and use `arrow` function when we use a callback, we think this will increase our code readability,
and that's actually what's `Kyle Simpson` the author of `you don't know js` recommended to do also the author of `Js the hard parts` for `Will Sentance` were doing in his course..

anyway, thanks @elyor-doniyorov , @D4nielJ for your informative reviews :100: